docs/neat_getlpaddrs.md

```diff
@@ -0,0 +1,32 @@
+# neat_getlpaddrs
+
+Obtains the local or peer addresses of a flow.
+
+### Syntax
+
+```c
+int neat_getlpaddrs(struct neat_ctx*  ctx,
+                    struct neat_flow* flow,
+                    struct sockaddr** addrs,
+                    const int         local)
+```
+
+### Parameters
+
+- **ctx**: Pointer to a NEAT context.
+- **flow**: Pointer to a NEAT flow.
+- **addrs**: Pointer to variable for storing pointer to addresses to.
```

Comment:
I would prefer if single-homing and multi-homing are handled in a similar way.
So not if the result is larger than 1 do this else this. It is better, IMHO, to have
one code path for error causes and one for the successful causes.
What do you think?